### ISSUE TITLE
[EN][NPC][HinterLands]Silvermane Stalker

### ISSUE THREAD

**Issue by TheRealestMcCoy** [dalaranwow/dalaran-wow#4624]
**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Currently has no name plates. This could be due to a issue they are suppose to be stealt and arent.
**Expected behaviour**: Tell us what should happen instead
They should be stealth and break stealth on combat and name plates should appear.
**Steps to reproduce the problem**:

1. Go to Hinter Lands 
2. Walk up on silvermane stalker
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
http://wotlk.openwow.com/npc=2926
http://www.wowhead.com/npc=2926/silvermane-stalker#comments
You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TheRealestMcCoy** [comment]
On a side note the name plates seem to be issues with alot of stealable npcs. I am unsure of all the names but some are in Maraudon, Dire Maul: East

**Comment by TeridanDW** [comment]
Hello, issue confirmed. The only relevent information regarding them, that i could find, is:

> WotLK Patch 3.2.0 (2009-08-04): Undocumented: These stalkers no longer stealth, to make finding them a lot easier, however they are black instead of grey as well.

Reference for them from legion - they don't have stealth and use the white model (same as the one used ingame now).
Fix for the wolves (they will appear as normal creatures now):
``` sql
UPDATE `creature_template_addon` SET `bytes1` = 0 WHERE `entry` = 2926; 
```
@TheRealestMcCoy  if you find more creatures like these, please report them, if you want to.
Thank.

**Comment by TheRealestMcCoy** [comment]
Ok well does that fix the name plates?

**Comment by TeridanDW** [comment]
Before:
![image](https://cloud.githubusercontent.com/assets/6797611/24213020/d581c85e-0f39-11e7-8e62-9410dab862d6.png)
After:
![image](https://cloud.githubusercontent.com/assets/6797611/24213053/ec7a8762-0f39-11e7-8353-6753f6e39cd9.png)

**Comment by TheRealestMcCoy** [comment]
Nice
@TeridanDW Check on these http://www.wowhead.com/npc=11792/putridus-shadowstalker

**Comment by Rushor** [comment]
if you remember the names of the creatures let me know, i will add it for the creature names you mentioned here

**Comment by TheRealestMcCoy** [comment]
@Rushor Tyring to remember and find them if i can kind sucks running back all over the places. Right now im in Wintersprings checking on a bear issue a friend told me about :)

**Comment by TheRealestMcCoy** [comment]
http://www.wowhead.com/npc=11456/wildspawn-shadowstalker

May be the ones in Dire Maul

**Comment by Rushor** [comment]
i already reworked the abilities of the stalker (with stealth) here https://github.com/dalaranwow/dalaran-wow/issues/3859

**Comment by Rushor** [comment]
DB/Creature: Silvermane Stalker - Enable nameplates by default

* closes: https://github.com/dalaranwow/dalaran-wow/issues/4624
* commit: 4814f9e5d2e73e8b41ca88dcb13e3f9531cb3b3f

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.


Thanks for the help

**Comment by Rushor** [comment]
http://www.wowhead.com/npc=11456/wildspawn-shadowstalker were also reworked with https://github.com/dalaranwow/dalaran-wow/issues/2503

**Comment by TeridanDW** [comment]
Dam, i am too slow :/ 
Both http://www.wowhead.com/npc=11792/putridus-shadowstalker and http://www.wowhead.com/npc=11456/wildspawn-shadowstalker are working correctly (with proper stealth spell and nameplates when they leave the stealth).

**Comment by TeridanDW** [comment]
Works as intended on PTR.
